packages/main/src/plugin/dockerode/libpod-dockerode.ts

```diff
@@ -165,6 +166,29 @@ export class LibpodDockerode {
       });
     };
 
+    // add pruneAllImages
+    prototypeOfDockerode.pruneAllImages = function () {
+      const optsf = {
+        path: '/v4.2.0/libpod/images/prune?all=true&', // this works
+        // For some reason the below doesn't work? TODO / help / fixme
```

Comment:
@benoitf any advice for this part? unsure what I'm doing wrong with the options / param call.

Reply:
only prune for pods should be added we should have the method on do ckerode

Reply:
Can't, doesn't work with dockerode.